easybuild/easyconfigs/p/Panoply/Panoply-5.6.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
```

`PackedBinary` is mostly interesting if you're using the custom `install_cmd` easyconfig parameter it supports.

That's not used here, so I would use `Tarball` easyblock instead

```suggestion
easyblock = 'Tarball'
```